docker-compose.yml

```diff
@@ -15,6 +15,11 @@ services:
         environment:
             API_URI: http://web:1080
             APP_SECRET:
+    pattern_library:
+        image: ${DOCKER_NAMESPACE}/pattern-library:${REVISION_PATTERN_LIBRARY}
+        healthcheck:
```

Comment:
healthcheck can be marked for removal now that it is copied upstream (but left in for now as the current `latest` image doesn't have it)